packages/kbn-storybook/storybook_config/webpack.config.js

```diff
@@ -95,6 +95,27 @@ module.exports = async ({ config }) => {
           },
         },
       },
+      {
+        loader: 'resolve-url-loader',
+        options: {
+          // If you don't have arguments (_, __) to the join function, the
+          // resolve-url-loader fails with a loader misconfiguration error.
+          //
+          // eslint-disable-next-line no-unused-vars
+          join: (_, __) => (uri, base) => {
+            if (!base || !parse(base).dir.includes('legacy')) {
+              return null;
+            }
+
+            // URIs on mixins in src/legacy/public/styles need to be resolved.
```

Comment:
any need for the src/legacy/ui/public rewrite?  not an issue, just thought i'd ask

Reply:
It wasn't affecting us in APM and.I don't know if anybody else is using it so I didn't copy it over.